utils/src/main/kotlin/Utils.kt

```diff
@@ -19,12 +19,24 @@
 
 package com.here.ort.utils
 
+import org.apache.logging.log4j.kotlin.KotlinLogger
+import org.apache.logging.log4j.kotlin.loggerOf
```

Comment:
What do you think about putting logging stuff into a separate file (inside this utils module)?

Reply:
Hmm... it's very little code. But why not. I'd then do the same for the CLI-related utilities in a separate commit.